File: CPP/Q17.cpp
#include <iostream>

int main() {
    char op = '+';
    int num1 = 10, num2 = 5;
    switch (op) {
        case '+': std::cout << num1 << " + " << num2 << " = " << num1 + num2 << std::endl; break;
        case '-': std::cout << num1 << " - " << num2 << " = " << num1 - num2 << std::endl; break;
        case '*': std::cout << num1 << " * " << num2 << " = " << num1 * num2 << std::endl; break;
        case '/': std::cout << num1 << " / " << num2 << " = " << num1 / num2 << std::endl; break;
        default: std::cout << "Invalid operator." << std::endl;
    }
    return 0;
}

File: Java/Q17.java
class Main {
    public static void main(String[] args) {
        char op = '+';
        int num1 = 10, num2 = 5;
        switch (op) {
            case '+': System.out.println(num1 + " + " + num2 + " = " + (num1 + num2)); break;
            case '-': System.out.println(num1 + " - " + num2 + " = " + (num1 - num2)); break;
            case '*': System.out.println(num1 + " * " + num2 + " = " + (num1 * num2)); break;
            case '/': System.out.println(num1 + " / " + num2 + " = " + (num1 / num2)); break;
            default: System.out.println("Invalid operator.");
        }
    }
}